doc/developers/contributing.rst

```diff
@@ -726,6 +726,16 @@ We are glad to accept any sort of documentation:
 
 .. dropdown:: Guidelines for writing docstrings
 
+  * You can use `pytest` to test docstrings, e.g. assuming the
+    `RandomForestClassifier` docstring has been modified, the following command
+    would test it's docstring compliance:
+
+    .. prompt:: bash
+
+      pytest --doctest-modules sklearn/ensemble/_forest.py -k RandomForestClassifier
```

Comment:
you should not need the `--doctest-modules` since it is already set in the `pyproject.toml` from the project, isn't it?

Reply:
At least in my setup I do seem to need the flag, but maybe I am missing something?

Reply:
Uhm, you are right. Apparently, it is something that we have in some other project (`skrub` or `imbalanced-learn`) but not in `scikit-learn`.

I assume it makes sense to not have it as a default because it might be to costly. So this proposal seems good then.